index.js

```diff
@@ -47,7 +47,7 @@ function parseTorrent (torrentId) {
   }
 }
 
-function parseTorrentRemote (torrentId, cb) {
+function parseTorrentRemote (torrentId, cb, options) {
```

Comment:
The callback argument should always be last. This is Node.js convention. You need to support it being called without options too, for convenience.